docs/guidance/stable-diffusion-rayservice.md

```diff
@@ -0,0 +1,55 @@
+# Serve a StableDiffusion text-to-image model using RayService
+
+> **Note:** The Python files for the Ray Serve application and its client are in the [ray-project/serve_config_examples](https://github.com/ray-project/serve_config_examples) repo 
+and [the Ray documentation](https://docs.ray.io/en/latest/serve/tutorials/stable-diffusion.html).
+
+## Step 1: Create a Kubernetes cluster with GPUs
+
+You can follow [aws-eks-gpu-cluster.md](./aws-eks-gpu-cluster.md) to create an AWS EKS cluster with 1
+CPU (`m5.xlarge`) and 1 GPU (`g5.xlarge`) nodes.
+
+## Step 2: Install KubeRay operator
+
+Follow [this document](../../helm-chart/kuberay-operator/README.md) to install the nightly KubeRay operator via 
+Helm. Note that the YAML file in Step 3 uses `serveConfigV2`, which is first supported by KubeRay v0.6.0.
```

Comment:
Can you move the "Note that the YAML..." sentence to Step 3 instead of keeping it here? Generally, I'd recommend keeping the notes in the same Steps that they're referring to. It's a bit hard to keep track of them all right now.

Reply:
I added this sentence to explain why I am asking users to install the nightly build instead of the latest stable release (v0.5.0). If we move this to Step 3, users might have already installed the wrong version of KubeRay. This sentence will be removed once v0.6.0 is released. Does it make sense to you?

Reply:
```suggestion
Helm. We're installing the nightly since the this example's YAML file uses `serveConfigV2`, which uses features that will be released in KubeRay v0.6.0.
```